zebra-chain/src/amount.rs

```diff
@@ -191,6 +191,54 @@ where
     }
 }
 
+impl<C> std::ops::Mul<i64> for Amount<C>
```

Comment:
Do we ever need to multiply by signed integers?

If not, let's change Mul and Div to u64:
```suggestion
impl<C> std::ops::Mul<u64> for Amount<C>
```

Reply:
Done as part of https://github.com/ZcashFoundation/zebra/pull/1051/commits/364f70eea3f19be9c35d884d861b5096018aa4fd